docs/templates/finops-hub-with-exports.md

```diff
@@ -0,0 +1,46 @@
+---
+QuickstartPath: tenant-deployments/finops-hub-with-exports
+---
+
+![Status: Not started](https://img.shields.io/badge/status-not%20started-red) &nbsp;<sup>→</sup>&nbsp;
+[![#1](https://img.shields.io/github/issues/detail/state/microsoft/cloud-hubs/1)](https://github.com/microsoft/cloud-hubs/issues/1)
+
+# FinOps hub with exports template
+
+This template creates a new **FinOps hub** instance and a Cost Management export to ingest cost details.
+
+FinOps hubs include Data Lake storage to host the export and a Data Factory instance with a pipeline to clean up duplicated cost data after each export is processed.
+
+On this page:
+
+- [Parameters](#parameters)
+- [Modules](#modules)
+- [Outputs](#outputs)
+
+---
+
+## Parameters
+
+- **subscription** (string) – ID of the subscription to deploy the hub instance to.
```

Comment:
I'm confused the description makes it sound like this template is the full export->adls->powerBI yet it lacks the parameters needed to create storage, tags, etc. like the other script.